src/components/TagList.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const TagList = (props) => {
+    return <ul className="c-tag-list">{props.children}</ul>;
+};
+
+TagList.propTypes = {
+    children: PropTypes.node,
```

Comment:
```suggestion
    children: PropTypes.node.isRequired,
```
Do we expect search results without any tags? (We may do now until the other tasks are finished, which is why I'm asking)

Reply:
Yes, that seems like a reasonable assumption